custom_components/isy994/entity.py

```diff
@@ -159,7 +159,7 @@ def device_state_attributes(self) -> Dict:
     async def send_node_command(self, command):
         """Respond to an entity service command call."""
         if not hasattr(self._node, command):
-            _LOGGER.error(
+            _LOGGER.exception(
                 "Invalid Service Call %s for device %s", command, self.entity_id
             )
```

Comment:
If its always going to called from a service (can't dig to check since I'm in flight), we usually let raise up so they can warn in the UI.

`raise HomeAssistantError(....)`

Reply:
Yes, only available from service calls. Fixed.